### DISCUSSION TITLE
Unable to access Nifi in cQube

### DISCUSSION THREAD

**Discussion by LavanyaKR3007** [Sunbird-cQube/community#37] [Q&A] [answered]
Hi @krishnakr6 , @SreenivasNimmagadda @jp-tibil 

Unable to access the Apache Nifi.

https://cqube.preprod.ntp.net.in/nifi - it shows 502 Bad Gateway Error

port 4200 is not listening 

Kindly provide your suggestions to fix the issue.

Please let me know if you need any information to support us,

cc: @srajasimman @Ramya-kmurthy

Regards,
Lavanya K R

**Comment by srajasimman** [comment]
Hi, @krishnakr6 @SreenivasNimmagadda @jp-tibil,

@LavanyaKR3007 found the issue and fixed it.

1. We have started the dashboard application using the PM2 daemon process manager `sudo pm2 resurrect` (resurrect previously dumped processes)

2. we can able to access the nifi from the VPN client system using the private IP and port [http://10.14.0.4:8096/nifi/](http://10.14.0.4:8096/nifi/)

Thanks to: @LavanyaKR3007

Regards,
@srajasimman

**Comment by LavanyaKR3007** [comment, marked as answer]
**Commands which solved the issued:**

sudo pm2 resurrect
sudo netstat -ntlp
sudo pm2 status

For opening Nifi , please find the below command for port Forwarding

sudo ssh -L 8096:10.14.0.4:8096 -A username@10.20.1.5

cc @srajasimman @Ramya-kmurthy

Thanks @jp-tibil @krishnakr6 for guidance

**Reply by SreenivasNimmagadda** [reply]
@LavanyaKR3007 
Thanks for the update

@jp-tibil and @krishnakr6 
Thanks for the quick help.
